lighty-modules/lighty-netconf-sb/src/main/java/io/lighty/modules/southbound/netconf/impl/NetconfClusteredTopologyPlugin.java

```diff
@@ -16,44 +16,67 @@
 import org.opendaylight.mdsal.dom.api.DOMMountPoint;
 import org.opendaylight.mdsal.dom.api.DOMMountPointService;
 import org.opendaylight.mdsal.dom.api.DOMRpcService;
+import org.opendaylight.mdsal.dom.api.DOMSchemaService;
 import org.opendaylight.netconf.client.NetconfClientDispatcher;
 import org.opendaylight.netconf.sal.connect.api.SchemaResourceManager;
 import org.opendaylight.netconf.sal.connect.impl.DefaultSchemaResourceManager;
+import org.opendaylight.netconf.sal.connect.netconf.DeviceActionFactoryImpl;
 import org.opendaylight.netconf.sal.connect.netconf.schema.mapping.DefaultBaseNetconfSchemas;
 import org.opendaylight.netconf.topology.singleton.impl.NetconfTopologyManager;
 import org.opendaylight.yang.gen.v1.urn.opendaylight.netconf.topology.singleton.config.rev170419.Config;
 import org.opendaylight.yang.gen.v1.urn.opendaylight.netconf.topology.singleton.config.rev170419.ConfigBuilder;
 import org.opendaylight.yang.gen.v1.urn.tbd.params.xml.ns.yang.network.topology.rev131021.NodeId;
 import org.opendaylight.yangtools.yang.common.Uint16;
 import org.opendaylight.yangtools.yang.data.api.YangInstanceIdentifier;
+import org.opendaylight.yangtools.yang.model.parser.api.YangParserException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class NetconfClusteredTopologyPlugin extends AbstractLightyModule implements NetconfSBPlugin {
 
-    private final NetconfTopologyManager topology;
+    private static final Logger LOG = LoggerFactory.getLogger(NetconfClusteredTopologyPlugin.class);
+
     private final DOMMountPointService domMountPointService;
+    private final LightyServices lightyServices;
+    private final String topologyId;
+    private final NetconfClientDispatcher clientDispatcher;
+    private final Integer writeTxIdleTimeout;
+    private final AAAEncryptionService encryptionService;
+    private NetconfTopologyManager topology;
```

Comment:
can be converted to local variable

Reply:
Nope, I don't think that will be a good idea to edit initProcedure() method. Do you see any other option how can be these convert to local variable?